includes/wizards/class-popups-wizard.php

```diff
@@ -383,27 +479,33 @@ public function enqueue_scripts_and_styles() {
 	/**
 	 * Get data to render Wizard.
 	 *
+	 * @param array|null $extras optional array of parameters to include in response.
 	 * @return WP_REST_Response
 	 */
-	public function api_get_settings() {
+	public function api_get_settings( $extras = [] ) {
 		$newspack_popups_configuration_manager = Configuration_Managers::configuration_manager_class_for_plugin_slug( 'newspack-popups' );
 
-		$response = [
-			'popups'   => [],
-			'segments' => [],
-			'settings' => [],
-		];
+		$response = array_merge(
+			[
+				'prompts'   => [],
+				'segments'  => [],
+				'campaigns' => [],
+				'settings'  => [],
+			],
+			$extras
```

Comment:
I added the casting in https://github.com/Automattic/newspack-plugin/pull/833/commits/57693f6e8a2246161d8155e9f4af26ae5563032a but I'm confused how that error would have occurred since all cases where `$extra` is used are explicitly arrays?

Reply:
That's hard to trace, but I did get that error logged when loading Campaigns Wizard.